apps/web/src/services/iam/components/UserManagementStatusModal.vue

```diff
@@ -59,13 +66,46 @@ const state = reactive({
     filteredServices: undefined,
     filteredItems: [],
     filteredUniqueItems: [],
+    selectedOnlyWorkspaceUsers,
```

Comment:
todo: Please assign default values to the `state` object! And include types if possible.